package.json

```diff
@@ -107,6 +107,7 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-import": "^2.26.0",
     "gaze": "^1.1.3",
+    "html-webpack-plugin": "^5.5.0",
```

Comment:
Do we need this? I couldn't see any reference to this plugin in this PR.

Reply:
Possible merging problem, I will fix in the next commit.